### DISCUSSION TITLE
VSCode Does Not Auto Complete SCSS Module Imports

### DISCUSSION THREAD

**Discussion by elliott-w** [vercel/next.js#13575] [Help]
# Bug report

## Describe the bug

VSCODE does not provide auto completion for SCSS module imports

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

`yarn create next-app` (use default and name it 'test-app')
`cd test-app`
`touch tsconfig.json`
`yarn dev` (CTRL+C after first run)
`yarn add --dev typescript @types/react @types/node`
`touch components/Button/index.tsx`
`touch components/Button/Button.module.scss`

Inside `components/Button/index.tsx` start typing `import './'

## Expected behavior

Button.module.scss appears under the auto-complete popup.

## Screenshots
![image](https://user-images.githubusercontent.com/30144036/83322787-e4c67d00-a29d-11ea-9029-222b81937b69.png)

![image](https://user-images.githubusercontent.com/30144036/83322568-176f7600-a29c-11ea-88b5-825c5e242038.png)

Only shows test.tsx

## System information

- OS: Windows 10
- Version of Next.js: 9.4.4
- Version of Node.js: v12.16.2
- Version of VSCode: 1.45.1

## Additional context

Adding
```
declare module '*.scss' {
    export const content: { [className: string]: string };
    export default content;
}
```
to next-env.d.ts does nothing

The 'SCSS Intellisense' vscode extension doesn't do anything either.

**Comment by juunzzi** [comment]
TT

**Reply by juunzzi** [reply]
I'm curious too TT

**Comment by root5427** [comment]
@juunzzi  any updates on this? Did you reach expected behavior?

**Comment by albertnev** [comment]
Same issue, very annoying since SCSS modules are usually the file with the largest name due to the `module.scss` part addition. Still not any solution?

**Comment by Revolt9k** [comment]
Any updates?

**Comment by TheCodemate** [comment]
Any solution for this issue?

**Comment by sburgpit** [comment]
Still silence ?

**Reply by TheCodemate** [reply]
Unfortunately the case is not solved. I thought it could be something with webpack... but anyway I havent fixed it

**Comment by samik3k** [comment]
Yep, im little frustrated when i cant autocomplete .scss module file

**Reply by TheCodemate** [reply]
That's sad :-( No one knows the answer

**Comment by bataevoff** [comment]
I am working with webpack and ts and I had the same issue using scss
I have downloaded an extension called "css modules" from vs code extensions store
It solved my problem

**Comment by ABC00012345** [comment]
same issue :((((

**Comment by cloudflips32** [comment]
Same issue for v1.82.0 and v1.81.0
![image](https://github.com/user-attachments/assets/78abc0f7-d430-449c-8898-661536de0c10)

Intellisense not recognizing heroSection.scss to import to HeroSection.jsx

**Reply by cloudflips32** [reply]
v1.81.0 and v1.82.0 of sass

**Reply by MrBns** [reply]
did you get any solution for this ?

**Comment by MrBns** [comment]
Bro, what is going on.
This might be simple issue going on for a long time. 

why it doesn't has any solution. in 2026 I am still getting this issue.
